DRCODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# This file defines which domain owns what parts of this repository.
+# This repository is a central place for defining Jenkins Jobs and Jarvis Suites to orchestrate
+# build and test workflows for pull requests and pipelines.  As such, it is expected
+# that this repository will have many owners and some shared areas (reusable macros, etc.)
+#
+# Please see DataRobot repo ./tools/code_owners.py tool to find which particular file
```

Comment:
Is this supposed to be present in public repos?

Reply:
Good point, removing it again